File: Euler9/Euler9.java
public class Euler9 {
     public static void main(String[] args){
        double c;
        for (int i = 1; i < 1000; i++) {
            for (int j = 1;  j < 1000; j++) {
                c = Math.sqrt((i*i)+(j*j)); //this is c
                if (i + j + c == 1000){
                    System.out.println((long) (i*j*c));
                    return;
                }
            }
        }
     }
}

File: Python/euler9.py
import math

def euler9():
    for i in range(1, 500):
        for j in range(1, 500):
            c = math.sqrt((i*i) + (j*j))
            if ((i + j + c) == 1000):
                result = int(i*j*c)
                print("Result: ", result)
                return

euler9()